### PR TITLE
updates to work with p2 and makeover the spawn page

### PR DESCRIPTION
**Pull request by rolyatmax**
This makes the spawn page look much nicer - updating it with a theme that matches jamsocket.com

### CODE REVIEW

**Review by paulgb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,58 +1,24 @@
...
-```
-
-## Jamsocket Guide
-
-For more information about the Jamsocket platform, check out our [docs](https://docs.jamsocket.com/).
+1. Fork this repository.
+2. Add your notebook(s) to the `/notebook` directory.
+3. Add your notebook's python dependencies to `requirements.txt`.
+4. Update the `NOTEBOOK` value in `run.sh` to make sure Jupyter opens your notebook on start up. 
+5. Add `JAMSOCKET_USER` and `JAMSOCKET_TOKEN` values to your GitHub secrets. (Contact us to get Jamsocket credentials.)
```
Comment (paulgb) [inline README.md]: should we make `Contact us` a `[mailto:hi@driftingin.space]` link?

Comment (rolyatmax) [reply README.md]: Yah - I'll add!

File: README.md [hunk 2]
```diff
@@ -1,58 +1,24 @@
...
+6. Push your repo to your GitHub account, and your Jupyter Notebook container will be automatically built and pushed to Jamsocket.
+7. Get a Spawn URL from us.
```
Comment (paulgb) [inline README.md]: “When users land on this URL, they'll be directed to a newly-spawned Jupyter notebook” or something?

File: notebook/example.ipynb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mJupyter cannot be started. Error attempting to locate jupyter: Running cells with 'Python 3.8.9 64-bit' requires jupyter and notebook package.\n",
```
Comment (paulgb) [inline notebook/example.ipynb]: Hmm, does this mean a traceback will be displayed when they open the notebook?

**Review by paulgb — APPROVED** [review]

File: notebook/example.ipynb (hunk 3 above)
Comment (rolyatmax) [reply notebook/example.ipynb]: Oh wait - I meant to delete this example.ipynb

**Pull request merged**